src/tests/daos_perf.c

```diff
@@ -1193,7 +1193,8 @@ main(int argc, char **argv)
 	if (ts_mode == TS_MODE_VOS) {
 		ts_ctx.tsc_cred_nr = -1; /* VOS can only support sync mode */
 		if (strlen(ts_pmem_file) == 0)
```

Comment:
if user specifies pmem_file with -f, then all ranks are going to write to the same file.
So we should probably do something like IOR, to also append the rank num to the file name even if it's specified by the user.

Reply:
aah ok sure will change it.